libs/login/feature/src/lib/login.component.scss

```diff
@@ -11,7 +11,7 @@ h2{
     text-transform: capitalize;
     position: relative;
     text-align: center;
-    margin-bottom: -40px;
+    margin-bottom: -60px;
```

Comment:
why are we using - margin?

Reply:
There is space above each field to allow for a label animation to take place, this negative margin compensates for the label so that the field is closer to the heading